proposals/cri-o.adoc

```diff
@@ -0,0 +1,220 @@
+== CRI-O
+
+*Name of project*: cri-o
+
+*Description*:
+
+CRI-O is a stable container runtime specifically tailored for Kubernetes.
+A founding principal was not "reinventing the wheel" and so the project uses shared components, refined approaches tested in production, and existing battle-tested code.
+Further, it is tuned for performance, stability, compatibility, and adherence to standards, particularly the Kubernetes Conformance tests.
+In short CRI-O is a building block of a Kubernetes cluster, to facilitate the life cycle of containers as required by the Kubernetes Container-Runtime-Interface (CRI).
+
+CRI-O was born in kubernetes-incubator and grew to production stable there.
+Some have claimed since being a kubernetes project that it is already "a CNCF project", but it is often left as an after thought in brand, awareness and support as a top-level project.
+This proposal is effective to promote from the kubernetes incubator a project that has fully matured.
+
+*Sponsor / Advisor from TOC*: Jonathan Boulle <jon@nstack.com>
+
+*Unique Identifier*: cri-o
+
+*License*: ALv2
+
+*Maturity Level*: Incubating
+
+*Source control repositories*: https://github.com/kubernetes-sigs/cri-o
+
+*Initial Committers*:
+
+- mrunalp
+- runcom
+- cyphar
+- mikebrow
+- sameo
+- rhatdan
+- nalind
+- giuseppe
+
+*Infrastructure requirements*: CI, CNCF Cluster, TravisCI
+
+*Issue tracker*: https://github.com/kubernetes-sigs/cri-o/issues
+
+*Mailing lists*
+
+* #cri-o on Freenode IRC
+* https://kubernetes.slack.com/messages/crio
+
+*Website*: https://cri-o.io
```

Comment:
I noticed on the website the branding says:

> Lightweight Container Runtime for Kubernetes

Would the change to the CNCF also be shifting it from one "for Kubernertes" to one that others can use for other purposes as well?

Reply:
if those other use cases _only_ conform to the kubernetes CRI, then _maybe_ but numero uno is k8s. :smile: